src/global.scss

```diff
@@ -277,6 +277,14 @@ $pendIconMarg: #{$baseUnit}px;
       padding-right: 5px;
     }
   }
+
+  iframe {
+    width: 100%;
+    min-height: 500px;
```

Comment:
Does it make sense to have `min-height` set here? Shouldn't that be handled by the `height` prop?

Reply:
I believe using `height` would override the `height=` property. My intention was to limit the smallest possible height to `500px`, while still allowing individual iframes to be larger if necessary.